metadata.go

```diff
@@ -69,15 +69,15 @@ type ImageSize struct {
 
 // ImageMetadata represents the basic metadata fields
 type ImageMetadata struct {
-	Orientation int
-	Channels    int
-	Alpha       bool
-	Profile     bool
-	Type        string
-	Space       string
-	Colourspace string
-	Size        ImageSize
-	EXIF        EXIF
+	Orientation    int
```

Comment:
Where does the formatting come from? Are there different styles in go or do people usually follow one common style or use one formatting tool?

Reply:
[This is the way.](https://blog.golang.org/gofmt) (`go fmt ...`)

So yes, another reason why Go code is pretty universally readable ... everyone uses the same formatting rules.